apps/reports/api_views/base_boundary.py

```diff
@@ -8,39 +8,408 @@ class BaseBoundaryReport(BaseReport):
+        "sambalpur"
+    ],
+    "kalahandi": [
+        "nuapada",
+        "balasore",
+        "boudh",
+        "kandhamal",
+        "rayagada",
+        "koraput",
+        "nabarangpur"
+    ],
+    "jagatsinghpur": [
+        "puri",
+        "cuttack",
+        "kendrapara"
+    ],
+    "sambalpur": [
+        "sonepur",
+        "boudh",
+        "angul",
+        "deogarh",
+        "sundergarh",
+        "jharsuguda",
+        "bargarh"
+    ],
+    "koraput": [
+        "malkangiri",
+        "nabarangpur",
+        "kalahandi",
+        "rayagada"
+    ],
+    "kandhamal": [
+        "rayagada",
+        "gajapati",
+        "ganjam",
+        "nayagarh",
+        "boudh",
+        "kalahandi"
+    ],
+    "khordha": [
+        "gajapati",
+        "nayagarh",
+        "cuttack",
+        "puri"
+    ],
+    "deogarh": [
+        "sundergarh",
+        "sambalpur",
+        "angul"
+    ],
+    "bhadrak": [
+        "keonjhar",
+        "balasore",
+        "jajpur",
+        "kendrapara"
+    ],
+    "nuapada": [
+        "kalahandi",
+        "bargarh",
+        "bolangir"
+    ],
+    "sonepur": [
+        "boudh",
+        "angul",
+        "sambalpur",
+        "bargarh",
+        "bolangir"
+    ],
+    "ganjam": [
+        "gajapati",
+        "kandhamal",
+        "nayagarh",
+        "khordha"
+    ],
+    "puri": [
+        "khordha",
+        "cuttack",
+        "jagatsinghpur"
+    ],
+    "dhenkanal": [
+        "angul",
+        "cuttack",
+        "jajpur",
+        "kendrapara"
+    ],
+    "gajapati": [
+        "rayagada",
+        "kandhamal",
+        "ganjam"
+    ],
+    "kendrapara": [
+        "bhadrak",
+        "jajpur",
+        "cuttack",
+        "jagatsinghpur"
+    ],
+    "sundergarh": [
+        "jharsuguda",
+        "sambalpur",
+        "deogarh",
+        "kendrapara",
+        "angul"
+    ],
+    "rayagada": [
+        "gajapati",
+        "kandhamal",
+        "koraput",
+        "kalahandi"
+    ],
+    "mayurbhanj": [
+        "balasore",
+        "kendrapara"
+    ],
+    "malkangiri": [
+        "koraput"
+    ],
+    "boudh": [
+        "bolangir",
+        "kalahandi",
+        "kandhamal",
+        "nayagarh",
+        "angul",
+        "sonepur",
+        "sambalpur"
+    ],
+    "jajpur": [
+        "keonjhar",
+        "cuttack",
+        "dhenkanal",
+        "kendrapara",
+        "bhadrak"
+    ],
+    "cuttack": [
+        "nayagarh",
+        "khordha",
+        "puri",
+        "jagatsinghpur",
+        "kendrapara",
+        "jajpur",
+        "dhenkanal",
+        "angul"
+    ],
+    "balasore": [
+        "keonjhar",
+        "bhadrak",
+        "mayurbhanj"
+    ],
+    "angul": [
+        "sonepur",
+        "boudh",
+        "nayagarh",
+        "cuttack",
+        "dhenkanal",
+        "keonjhar",
+        "sundergarh",
+        "deogarh",
+        "sambalpur"
+    ],
+    "nabarangpur": [
+        "koraput",
+        "rayagada",
+        "kalahandi"
+    ]
+    }
+
```

Comment:
This `OLP_neighbor_names` was manually mapped by looking into the districts at http://www.mapsofindia.com/maps/orissa/orissa.htm. With this, the `neighbourIds` can be calculated.

This was ideally supposed to be calculated from the electedrep table. But its origins for KLP is lost in time as of writing this comment.